#goal
float32 x
float32 y
float32 z
float32 or_x
float32 or_y
float32 or_z
float32 or_w
---
#result
int32 n_obstacles
float32[] obst_coords
---
#feedback
int32 current_state
